docs/source/models/intervention_models/neonatal/antibiotics_intervention.rst

```diff
@@ -0,0 +1,131 @@
+    - 
+  * - CEmONC
+    - Comprehensive emergency obstetric and neonatal care
+    - 
+
+Intervention Overview
+-----------------------
+
+Antibiotics are used to treat bacterial infections in neonates, reducing the risk mortality.
+
+This section describes how an antibiotic-treatment intervention can be implemented and calibrated for the :ref:`MNCNH Portfolio model <2024_concept_model_vivarium_mncnh_portfolio>`.
+
+.. list-table:: Affected Outcomes
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Modeled?
+    - Note (ex: is this relationship direct or mediated?)
+  * - Neonatal sepsis and other neonatal infections Mortality Probability
+    - Decrease multiplicatively
+    - Yes
+    - 
+
+Baseline Coverage Data
+++++++++++++++++++++++++
+
+X% of CEmONC facilities and Y% of BEmONC facilities have CPAP (check if this is included in the 2016 Ethiopia EmONC Final Report).  Please use placeholder values for now while we try to find reliable values.  We might be able to borrow strength from other locations and times by predicting coverage more country-years simultaneously, perhaps even in combination with other key intervention technologies, based on sources such as existing `Service Provision Assessment (SPA) <https://www.dhsprogram.com/methodology/Survey-Types/SPA.cfm>` and `Service Availability and Readiness Assessment (SARA) <https://www.who.int/data/data-collection-tools/service-availability-and-readiness-assessment-(sara)>` data.
+
+Vivarium Modeling Strategy
+--------------------------
+
+This intervention requires adding an attribute to all simulants to specify if a birth happens in a facility with access to antibiotics.  Since the neonatal mortality model does not explicitly represent incidence of sepsis, we will also not track who receives antibiotics.  Instead the model will have different cause-specific mortality rates for sepsis for individuals born with and without access to antibiotics.
+
+We will use the decision tree below to find the probability of sepsis mortality with and without access to antibiotics that are logically consistent with the baseline delivery facility rates and baseline antibiotics coverage (and population level sepsis mortality probability). Since the neonatal model includes different mortality probabilities for both the early and late neonatal periods, this calculation must be performed for both time periods as well.
+
+We will then add an attribute to each simulant indicating whether the birth occurs at a facility with access to antibiotics (which will be dependent on the facility choice, i.e. no access for home births and lower access for BEmONC than CEmONC facilities).
+
+We will then use the conditional probabilities for simulants with and without access to determine which simulants die from sepsis.
+
+While we are searching the literature, we will use a placeholder relative risk of 0.50 (95% CI 0.25-0.75) of sepsis mortality for neonates with access to antibiotics.
```

Comment:
This contradicts the placeholder of 0.72 in https://github.com/ihmeuw/vivarium_research/pull/1588